src/pages/settings/Security/TwoFactorAuth/Steps/CodesStep.js

```diff
@@ -0,0 +1,146 @@
+import React, {useEffect, useState} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {ActivityIndicator, View} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import * as Expensicons from '../../../../../components/Icon/Expensicons';
+import withLocalize, {withLocalizePropTypes} from '../../../../../components/withLocalize';
+import compose from '../../../../../libs/compose';
+import * as Illustrations from '../../../../../components/Icon/Illustrations';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../../../components/withWindowDimensions';
+import styles from '../../../../../styles/styles';
+import FixedFooter from '../../../../../components/FixedFooter';
+import Button from '../../../../../components/Button';
+import PressableWithDelayToggle from '../../../../../components/Pressable/PressableWithDelayToggle';
+import Text from '../../../../../components/Text';
+import Section from '../../../../../components/Section';
+import ONYXKEYS from '../../../../../ONYXKEYS';
+import Clipboard from '../../../../../libs/Clipboard';
+import themeColors from '../../../../../styles/themes/default';
+import localFileDownload from '../../../../../libs/localFileDownload';
+import * as Session from "../../../../../libs/actions/Session";
+import StepWrapper from "../StepWrapper/StepWrapper";
+import CONST from "../../../../../CONST";
+
+const propTypes = {
+    ...windowDimensionsPropTypes,
+    ...withLocalizePropTypes,
+    account: PropTypes.shape({
+        /** User recovery codes for setting up 2-FA */
+        recoveryCodes: PropTypes.string,
+
+        /** If recovery codes are loading */
+        isLoading: PropTypes.bool,
+    }),
+
+    /** Method to set the next step */
+    setStep: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
```

Comment:
Use prop destructuring with default values instead of `defaultProps` -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring